tests/test_vpc.rb

```diff
@@ -87,7 +88,7 @@ def test_auto_recovery_reload_delay
     assert_equal(300, @vpc.auto_recovery_reload_delay,
                  'Auto recovery delay should be 300')
     # negative high range
-    e = assert_raises(CliError) { @vpc.auto_recovery_reload_delay = 3601 }
+    e = assert_raises(CliError) { @vpc.auto_recovery_reload_delay = 86_401 }
```

Comment:
This is just part of another test. I just re-ran all the mini-tests. If you want to remove these lines. I can do it.